bpf/map_sizing.h

```diff
@@ -3,7 +3,7 @@
 
 // TODO: make this user-configurable and modify the value from the userspace when
 // loading the maps with the Cilium library
-#define MAX_CONCURRENT_REQUESTS 1000 // 1000 requests per second max for a single traced process
-#define MAX_CONCURRENT_SHARED_REQUESTS 10000 // 10 * MAX_CONCURRENT_REQUESTS total ongoing requests, for maps shared among multiple tracers, e.g. pinned maps
+#define MAX_CONCURRENT_REQUESTS 10000 // 10000 requests per second max for a single traced process
+#define MAX_CONCURRENT_SHARED_REQUESTS 30000 // 10 * MAX_CONCURRENT_REQUESTS total ongoing requests, for maps shared among multiple tracers, e.g. pinned maps
```

Comment:
We could address this as another task, for other maps too: you can move the map sizing from compile-time to runtime and let the user decide the size (this way we could troubleshoot better).

This is how the Network Metrics side does it at the user side, from the user configuration:

https://github.com/grafana/beyla/blob/main/pkg/internal/netolly/ebpf/tracer.go#L85-L86

The aggregated_flows map does not define any size when it is compiled:

https://github.com/grafana/beyla/blob/main/bpf/flows.c#L56-L60

Reply:
Ah great point, I'll follow-up with a configuration option. 1000 was way too small, too many collisions on my 20 core machine.